### ISSUE TITLE
Request mod update for crazy craft, and maxi stacks

### ISSUE THREAD

**Issue by JxffMxrris** [ckdextergames/Icarus_mods#3]
Good morning! I was wondering if you had planned to update crazy craft and maxi stack mods for Icarus for week 70? Both seem to not be working with the new update or maybe I'm just doing something wrong lol. No rush and thank you in advance! Have a great day(:

**Comment by ckdextergames** [comment]
As long as after the game updated you told Icarus Mod Manager (IMM) to update its data folder and then you re-installed your mods, you should be fine. Any changes should be accounted for.

Hopefully you do this each week, since many of us no longer re-create and re-upload our mods each week any more. It would be just way too much work.

If you’re not using IMM, then yes I’d have to update those for you specially. The advantage of IMM is that even if you’re just using one single mod, it lets you make sure that that mod will be revised if /as needed to account for any changes in that week’s game build — without having to rely on that mod’s creator to re create and repost all their mods every week.

Happy to help if needed. To reach me faster, DM me in the Discord.
Cheers!
________________________________
From: JxffMxrris ***@***.***>
Sent: Friday, April 7, 2023 6:27 AM
To: ckdextergames/Icarus_mods ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ckdextergames/Icarus_mods] Request mod update for crazy craft, and maxi stacks (Issue #3)


Good morning! I was wondering if you had planned to update crazy craft and maxi stack mods for Icarus for week 70? Both seem to not be working with the new update or maybe I'm just doing something wrong lol. No rush and thank you in advance! Have a great day(:

**Comment by JxffMxrris** [comment]
Yeah I did that and it still wasn't working so more than likely, user error lol. But awesome! I'll look into and double check and see what I did wrong, thanks for the quick response! I'll join the discord as well thank you for the heads up! You're the man🤘🏼🤘🏼
